src/applications/user-testing/disability-benefits-all-claims-conditions-chapter/content/conditions.jsx

```diff
@@ -29,28 +29,45 @@ export const NewConditionDescription = () => (
   </>
 );
 
-const createSecondaryDescriptionString = causedByCondition => {
-  // Just for SecondaryEnhanced demo
-  if (typeof causedByCondition === 'object') {
-    return createSecondaryEnhancedDescriptionString(causedByCondition);
-  }
+const createCauseFollowUpDescriptions = (item, fullData = {}) => {
+  if (!item?.cause) return '';
 
-  return `caused by ${causedByCondition ||
-    'a missing service-connected condition'}`;
-};
+  const { conditions = [], ratedDisabilities = [] } = fullData;
+
+  switch (item.cause) {
+    case 'NEW':
+      return 'caused by an injury, event, disease or exposure during my service';
+
+    case 'SECONDARY': {
+      if (typeof item.causedByCondition === 'object') {
+        return createSecondaryEnhancedDescriptionString(
+          item.causedByCondition,
+          fullData,
+        );
+      }
+
+      const target = item.causedByCondition?.toLowerCase()?.trim();
+      const foundInConditions = conditions.some(
+        c => c.newCondition?.toLowerCase()?.trim() === target,
```

Comment:
We should have some shared utility function that normalizes this since it's a common pattern: `const normalize = str => str?.toLowerCase()?.trim();`

Reply:
Normalizing strings is definitely reusable. For the prototype I kept it inline to avoid pulling in new helpers. It's likely we'll extract a normalize() util when we merge this into the main codebase.